cmd/server.go

```diff
@@ -128,6 +133,19 @@ func createHTTPServer(ctx api.Context) *echo.Echo {
 	e.Use(middleware.Logger())
 	e.Use(ServerCache)
 
+	// Set UserID in trace headers
+	e.Use(func(next echo.HandlerFunc) echo.HandlerFunc {
+		return func(c echo.Context) error {
+			if userID := c.Request().Header.Get(api.UserIDHeaderKey); userID != "" {
+				span := trace.SpanFromContext(c.Request().Context())
+				span.SetAttributes(
+					attribute.String("user-id", userID),
```

Comment:
I think we should add org and clerk id in here as well

Reply:
👍🏽